### DISCUSSION TITLE
Any way to modify code from Sensor Watch Builder?

### DISCUSSION THREAD

**Discussion by bigjas465** [joeycastillo/Sensor-Watch#400] [Q&A]
Is there a way to modify the code that is built with the online Sensor Watch Builder? I would like to change the sound option to something custom (the ubiquitous "doorbell" sound) but still keep the customized watch faces. My hope is that it would be easy to change the music commands for the Mario or Zelda tunes but keep everything else the same.

The builder gives an output file in UF2 format, which I understand is not easily editable. Is there another way to edit the code? I am not a programmer but I am confident that I can alter a few commands if I knew how to access an editable file.

Thank you in advance.

**Comment by wryun** [comment]
No, the builder has no current way to modify the code. If you want to do this you'll have to bite the bullet and do the compilation yourself.

The file you'd need to edit is: https://github.com/joeycastillo/Sensor-Watch/blob/439843f56b1cd6f8976b2ecabb27457a3b3e233c/movement/movement_custom_signal_tunes.h

**Reply by bigjas465** [reply]
Thank you for the reply. I am already into the files and it seems pretty straightforward. I might throw in a bit of Bach while I am at it.
